api/Cache.json

```diff
@@ -0,0 +1,475 @@
+{
+  "api": {
+    "Cache": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Cache",
+        "support": {
+          "chrome": {
+            "version_added": "40",
+            "notes": [
+              "Available on global scope from version 43"
+            ]
+          },
+          "edge": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "39",
+            "notes": [
+              "Service workers (and <a href='https://developer.mozilla.org/docs/Web/API/Push_API'>Push</a>) have been disabled in the <a href='https://www.mozilla.org/firefox/organizations/'>Firefox 45 & 52 Extended Support Releases</a> (ESR.)"
+            ]
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "27",
+            "notes": [
+              "Available on global scope from version 30"
+            ]
+          },
+          "safari": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": "40",
+            "notes": [
+              "Available on global scope from version 43"
+            ]
+          },
+          "chrome_android": {
+            "version_added": "40",
+            "notes": [
+              "Available on global scope from version 43"
+            ]
+          },
+          "firefox_android": {
+            "version_added": "39"
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": "27",
+            "notes": [
+              "Available on global scope from version 30"
+            ]
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "add": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Cache/add",
+          "support": {
+            "chrome": {
+              "version_added": "44",
+              "notes": [
+                "Require HTTPS from version 46"
```

Comment:
'Requires' not 'require'. Please fix throughout.

Reply:
Oops, that's what I originally thought. Just fixed that! Thanks